pkg/main.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
I would rather this live at `cmd/cleaner/main.go` and `cmd/k8ssandra-client/main.go`.

Reply:
There's supposed to be only one main.go.

Reply:
I am basing this off of https://github.com/golang-standards/project-layout.

Reply:
Even if we only have a single hook/job, I am not a fan of having a `main.go` under the `pkg` dir.

Reply:
I can move it to under cmd